cocos/tween/tween.ts

```diff
@@ -79,6 +80,20 @@ export class Tween<T extends object = any> {
         return this;
     }
 
+    /**
+     * @en Set the id for previous action
+     * @zh 设置前一个动作的 id
+     * @param id @en The internal action id to set @zh 内部动作的 id 标识，
+     * @note @en Don't pass in a value -1 if you want to mark the previous action since -1 is the default value
+     *       @zh 如果你想标识前一个动作，请勿传递 -1，因为 -1 是所有动作的默认标识
```

Comment:
I think -1 is a invalid value, don't have to support it here.

Reply:
Fixed.